task/pnc-prebuild-git-clone-oci-ta/0.1/patch.yaml

```diff
@@ -0,0 +1,76 @@
+- op: replace
+  path: /metadata/name
+  value: pnc-prebuild-git-clone-oci-ta
+- op: replace
+  path: /metadata/annotations/tekton.dev~1displayName
+  value: pnc prebuild git clone oci trusted artifacts
+- op: replace
+  path: /spec/description
+  value: The pnc-prebuild-git-clone-oci-ta Task will clone a repo from the provided url, apply
+    PNC prebuild modifications and store it as a trusted artifact in the provided OCI repository.
+
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_SCRIPT
+    description: The build script to embed with the Containerfile
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_TOOL
+    description: The build tool to use (ant, gradle, maven, sbt).
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_TOOL_VERSION
+    description: The build tool version to use (e.g. 3.9.5)
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: JAVA_VERSION
+    description: Java version to use (7, 8, 9, 11, 17, 21, 22, 23)
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: PNC_KONFLUX_TOOLING_IMAGE
+    description: Name of the tooling image.
+    type: string
```

Comment:
How often is this going to change? How generic is the image to satisfy a majority of use cases? It seems like it would be better to bake this type of image reference in to a task itself instead of requiring users to define it for each of their onboarded components' pipeline runs.

Reply:
Actually thats a good point ; now we have extracted the code from JBS into a onboarded repository that parameter/value could probably be removed and hardcoded to something like e.g. `quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling:latest`